helm-swoop.el

```diff
@@ -150,6 +150,9 @@
     (switch-to-buffer $buf))
   "Change the way to split window only when `helm-swoop' is calling")
 
+(defvar helm-swoop--goto-line-action-hook nil
```

Comment:
`--` which means private variable/function should not be used if it is used by users, not developers. And then `defcustom` is better than `defvar`